packages/valory/skills/abstract_round_abci/handlers.py

```diff
@@ -681,12 +715,37 @@ def _handle_response(
         self.context.logger.info(f"{log_message}: {message}")
         dialogues = cast(TendermintDialogues, self.dialogues)
         dialogues.dialogue_stats.add_dialogue_endstate(
-            TendermintDialogue.EndState.CONFIG_SHARED, dialogue.is_self_initiated
+            TendermintDialogue.EndState.COMMUNICATED, dialogue.is_self_initiated
         )
 
-    def _handle_error(
+    def _response_recovery_params(
         self, message: TendermintMessage, dialogue: TendermintDialogue
     ) -> None:
+        """Process params-sharing response messages."""
+
+        if not self._check_registered(message, dialogue):
+            return
+
+        try:
+            recovery_params = json.loads(message.params)
+            shared_state = cast(SharedState, self.context.state)
+            shared_state.tm_recovery_params = TendermintRecoveryParams(
+                **recovery_params
```

Comment:
don't we care about the sender?

Reply:
Correct, though I will include this on another PR, which will store all the responses on the shared state and will also have functionality for choosing the most common one.

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1736#discussion_r1067309643